stl/inc/format

```diff
@@ -3285,9 +3285,11 @@ void _Range_formatter_format_as_string(_Range&& _Rng, _FormatContext& _Ctx, cons
     if constexpr (_RANGES contiguous_range<_Range>) {
         const auto _Size = _STD _To_unsigned_like(_RANGES distance(_Rng));
 
-        if (!_STD in_range<size_t>(_Size)) [[unlikely]] {
-            _Throw_format_error("Formatted range is too long.");
+#if _CONTAINER_DEBUG_LEVEL > 0
+        if constexpr (sizeof(_Size) > sizeof(size_t)) {
+            _STL_VERIFY(_Size <= SIZE_MAX, "The size of the formatted range is too large for a valid range.");

```

Comment:
I think `PTRDIFF_MAX` is realistically the upper bound on the number of elements in a single array in C++ in order for `last - first == n` to be well-defined. I also find it weird that a distance of `-2` might be reported as "too large":
```suggestion
            _STL_VERIFY(_Size <= size_t{PTRDIFF_MAX}, "contiguous range has impossible size.");
```